Playground/RxSwiftExtPlayground.playground/Pages/weak.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,49 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+ 
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import Foundation
+import RxSwift
+import RxSwiftExt
+
+private class Tool {
+    
+    deinit {
+        print("deinit")
+    }
+    
+    func display(_ value: Any) {
+        print(value)
+    }
+    
+    func produce(_ value: Int) -> Observable<Int> {
+        return Observable.of(value * 2, value * 3)
+    }
+}
+
+example("weak") {
```

Comment:
I wonder if we should make a page for each of these operators separately instead of sticking them in a single page. We usually have a page-per-operator, it seems.

Reply:
done